modules/snowflake_connector_config/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  connector_name = "${var.environment_name}-snowflake-connector"
+  kafka_topics   = "opscore.qa"
+}
+
+data "template_file" "snowflake_connector_config" {
+  template = file("${path.module}/snowflake-connector-config.tpl")
+
+  vars = {
+    connector_name                   = local.connection_name
+    kafka_topics                     = local.kafka_topics
+    snowflake_database               = var.snowflake_database
+    snowflake_private_key            = var.snowflake_private_key
+    snowflake_private_key_passphrase = var.snowflake_private_key_passphrase
+    snowflake_topic2table            = var.snowflake_topic2table
+    snowflake_url                    = var.snowflake_url
+    snowflake_username               = var.snowflake_username
+  }
+}
+
+resource "local_file" "json" {
+  sensitive_content = data.template_file.snowflake_connector_config.rendered
+  filename          = "${path.module}/snowflake-connector-config.json"
+}
+
+resource "null_resource" "upload_config" {
+  triggers = {
+    cluster_url     = var.kafka_connect_url
+    connection_name = var.connection_name
```

Comment:
OK, so this won't change if the config changes.
There's probably not a great way to make that happen, and it's not hard to `tg rm --target ...`.

At some point, I want to migrate to a better provider anyways.